tests/location_tests.cpp

```diff
@@ -0,0 +1,29 @@
+/// @copyright See `LICENSE` in the root directory of this project.
+
+#include <antler/project/location.hpp>
+
+#include <catch2/catch.hpp>
+
+TEST_CASE("Testing location clone") {
+   using namespace antler::project;
+
+   antler::system::fs::remove_all("./foo");
+
+   location::clone_github_repo("antelopeio", "antler-proj", "main", 10, "./foo/foo2");
+   CHECK(antler::system::fs::exists("./foo/foo2/.git"));
```

Comment:
antler-proj/tests/location_tests.cpp:13: FAILED:
  CHECK( antler::system::fs::exists("./foo/foo2/.git") )
with expansion:
  false

Reply:
This is passing for me. I'm not sure what, if anything, changed.

Reply:
I tried to play with it, so if I run it standalone - it passes, if I run it in parallel, it fails. It seems that it used same directories as other tests so I fixed it by renaming test project directory